documents/templates/commcomm_to_tsc.md

```diff
@@ -0,0 +1,7 @@
+```
```

Comment:
TBH we don't need, and I don't think we should have, templates for this. Each move is going to be pretty one off, and the _actual_ steps involved will vary from transfer to transfer. Using a template encourages fitting a square peg in a round hole.